app/controllers/insured/fdsh_ridp_verifications_controller.rb

```diff
@@ -17,11 +18,16 @@ def new
       end
     end
 
-    def wait_for_primary_response; end
+    def wait_for_primary_response
+      authorize @person, :complete_ridp?
```

Comment:
I thought we were doing custom policy methods per controller action, this one is just reusing _complete_ridp?_

Reply:
The controller actions are using 'complete_ridp?' because having matching controller actions doesn't make sense in this case.

Let's consider why:
1. The sole purpose of all actions in this controller is to execute the RIDP flow.  All actions are the equivalent of an 'update' - but performed specifically on ridp data under person.
2. The object on which we are performing the policy enforcement is a Person, not some abstract RIDP object.
3. Because the methods are in the person policy, having one method which exactly matches each controller action would make little sense.  Let's imagine we *did* do that.  Can you imagine what the person policy would look like full of 'wait_for_primary_response?', 'wait_for_secondary_response?', 'create?' that actually apply to RIDP?  Now, just for the sake of consistency, we have the policy littered with names that make no sense.
4. The reason I'm arguing for breaking the convention here is because the controller *already* breaks the convention - and we don't have the time to refactor it.